impl/client/src/doc/javascript/template/vizApi/tmpl/type.tmpl

```diff
@@ -1,7 +1,207 @@
 <?js
-    var data = obj;
-    var self = this;
-    data.forEach(function(name, i) { ?>
-<span class="param-type"><?js= self.linkto(name, self.htmlsafe(name)) ?></span>
-<?js if (i < data.length-1) { ?>|<?js } ?>
-<?js }); ?>
\ No newline at end of file
+  var data = obj;
```

Comment:
Still can't find all the types in the original `registerStandardJsTypes`.
Also, still this whole code is still in a tmpl file...